### PR TITLE
Contest bearing

### PR DESCRIPTION
**Pull request by int2001**
Added Bearing-Lookup to contesting.

image

### CODE REVIEW

**Review by phl0 — CHANGES_REQUESTED** [review]
Bearing is not cleared upon reset / logging the QSO. Also not updated when logging a new station.

File: application/views/contesting/index.php [hunk 1]
```diff
@@ -103,7 +103,8 @@
                             <div class="mb-3 col-md-3">
                                 <label for="callsign"><?php echo lang('gen_hamradio_callsign'); ?></label>
                                 <input type="text" class="form-control form-control-sm" id="callsign" name="callsign" required>
-                                <small id="callsign_info" class="badge text-bg-danger"></small>
+                                <small id="callsign_info" class="badge text-bg-danger"></small><br/>
```
Comment (HB9HIL) [inline application/views/contesting/index.php:106]: just remote the `<br>`. everything else is great :) thx for that one! 

```suggestion
                                <small id="callsign_info" class="badge text-bg-danger"></small>
```

**Review by HB9HIL — APPROVED** [review]

File: application/views/contesting/index.php (hunk 1 above)
Comment (HB9HIL) [reply application/views/contesting/index.php:106]: rr was able to reproduce that. sorry

**Review by phl0 — APPROVED** [review]
Works.

File: application/views/contesting/index.php (hunk 1 above)
Comment (int2001) [reply application/views/contesting/index.php:106]: as chatted, @HB9HIL - if you have an idea how to fix that (and if its important enough), pse open new PR.
I tested with Safari, FF and Chrome (wothout BR) and the result was always bearing direct next to Warning.

As best-effort i'll merge this.

**Pull request merged**